foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/ConfigStoreInit.java

```diff
@@ -52,7 +67,12 @@ public void init() {
         logger.info("Using property Dao class: {}", propertyDaoClass);
 
         try {
-            ConfigLoader configLoader = CDI.current().select(ConfigLoader.class,new Any.Literal()).get();
+            // Pull the original configs onto the device
+            // Because the configs are read only, these copied files are loaded into the config-store and manipulated
+            // if needed.
+            pullConfigs();
```

Comment:
Q: If I understand correctly before you could configure if you wanted to skip loading configures during initialization. If that's right was this feature removed?

Reply:
It was removed. Before, the original configs were read and write. This means we didnt need to pull them off the volume because we could interact with them there. But now that its read only, we need our own that we can manipulate. Only comes into play for the light weight solution as the others were already saving configs off somewhere else.